package.json

```diff
@@ -23,10 +23,11 @@
     "js-priority-queue": "^0.1.5",
     "jszip": "^2.5.0",
     "paralleljs": "^0.2.1",
-    "three": "^0.77.0",
+    "simd": "^2.0.0",
     "string_format": "^0.0.6",
-    "when": "^3.7.7",
-    "simd": "^2.0.0"
+    "three": "^0.77.0",
+    "whatwg-fetch": "^1.0.0",
```

Comment:
We need to use a promise polyfill too. `when` comes with one but it needs to be explicitly invoked AFAICT (possibly just adding the appropriate entry to webpack's `entry`).
See also https://github.com/iTowns/itowns2/pull/95#issuecomment-230220345